### PR TITLE
Feedback evaluation detaching bug fixing 🙉🍌

### PR DESCRIPTION
**Pull request by Evgeniy-Sinicin**

### CODE REVIEW

File: InternshipBE/DAL/Repositories/FeedbackRepository.cs [hunk 1]
```diff
@@ -40,5 +40,57 @@ public async Task<List<Feedback>> GetFeedbacksByCandidateIdAsync(int id)
...
+                var evaluationsToDelelte = oldEvaluations.Where(x => evaluationIdsToDelelte.Contains(x.Id));
+                _context.Evaluations.RemoveRange(evaluationsToDelelte);
+
+                evaluations.ForEach(x => feedback.Evaluations.Add(x));
+            }
+
+            _context.Entry(feedback).State = EntityState.Modified;
+            await _context.SaveChangesAsync();
+
+            return feedback;
```
Comment (romanstat) [inline InternshipBE/DAL/Repositories/FeedbackRepository.cs]: жесть, try to optimize/refactor

**Review by romanstat — APPROVED** [review]

**Review by Xodoc — APPROVED** [review]

**Review by ihhyy — APPROVED** [review]

File: InternshipBE/DAL/Validators/EvaluationValidator.cs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+﻿using DAL.Entities;
+
+namespace DAL.Validators
+{
+    public class EvaluationValidator : Validator<Evaluation>
+    {
+
```
Comment (Xodoc) [inline InternshipBE/DAL/Validators/EvaluationValidator.cs:7]: unnecessary space

Comment (Evgeniy-Sinicin) [reply InternshipBE/DAL/Validators/EvaluationValidator.cs:7]: It is git illusion. ✨
Unnecessary space is not exist in real project. 🙃

File: InternshipBE/DAL/Repositories/EvaluationRepository.cs [hunk 3]
```diff
@@ -23,5 +26,26 @@ public async Task<List<Evaluation>> GetEvaluationsByFeedbackId(int feedbackId)
                 .Where(x => x.FeedbackId == feedbackId)
                 .ToListAsync();
         }
+
+        public async Task<int> DeleteMissingEvaluationsByFeedbackId(int feedbackId, List<Evaluation> evaluations)
+        {
+            var evaluationsToDelete = await _context.Evaluations
+                .Include(x => x.Feedback)
+                .Include(x => x.Skill)
```
Comment (romanstat) [inline InternshipBE/DAL/Repositories/EvaluationRepository.cs]: ?

**Review by romanstat — APPROVED** [review]

**Pull request merged**